File: hoosong0235/Silver/2581. 소수/소수.py
primes = []

M = int(input())
N = int(input())

for i in range(M, N+1):
    is_prime = True
    for j in range(2, int(i**0.5)+1):
        if (i%j == 0):
            is_prime = False
    if (is_prime and i!=1):
        primes.append(i)
        
if (len(primes) == 0):
    print(-1)
else:
    print(sum(primes))
    print(min(primes))

File: hoosong0235c/Silver/2581. 소수/소수.c
#include <stdio.h>

int solve(int);

int main(void) {
    int m, n, min = -1, sum = 0;
    scanf("%i\n%i", &m, &n);
    for (int i = m; i <= n; i++) if (solve(i)) { if (min == -1) min = i; sum += i; }
    if (min == -1) printf("-1");
    else printf("%i\n%i", sum, min);

    return 0;
}

int solve(int n) {
    if (n == 1) return 0;
    for (int i = 2; i < n; i++) if (n % i == 0) return 0;
    return 1;
}